daemon/inspect_darwin.go

```diff
@@ -0,0 +1,30 @@
+// +build darwin
```

Comment:
rename _darwin => _unix would be appropriate.
you dont have to have // +build line if the filename tells that.